Source/ACE.Server/Managers/EmoteManager.cs

```diff
@@ -870,7 +870,8 @@ public float ExecuteEmote(BiotaPropertiesEmote emoteSet, BiotaPropertiesEmoteAct
                     break;
 
                 case EmoteType.SetBoolStat:
-                    targetObject.SetProperty((PropertyBool)emote.Stat, emote.Amount == 0 ? false : true);
+                    player.UpdateProperty(player, (PropertyBool)emote.Stat, emote.Amount == 0 ? false : true);
```

Comment:
this can't be assumed that the targetObject is always a player here... targetObject should never be null, but it's possible the targetObject could not be a player